src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/TokenVerifyingConfigFactory.kt

```diff
@@ -18,40 +18,44 @@ class TokenVerifyingConfigFactory(
                 WebTokenHelper.instance.verifier,
                 TokenStore.instance
         )
+        val githubDirectClientWrapper = GithubDirectClientWrapper()
     }
 
     val clientWrappers = mutableListOf(headerClientWrapper, cookieClientWrapper)
 
     override fun build(vararg parameters: Any?): Config
     {
-        clientWrappers.forEach {
-            it.client.addAuthorizationGenerator({ _, profile -> extractPermissionsFromToken(profile) })
```

Comment:
I've moved things around so that clients can specify their own `AuthorizationGenerator`s, in particualr so that the Github client, which doesn't have a JWT token, can add permissions in a different way.